src/components/side-menu/side-menu.component.ts

```diff
@@ -39,8 +39,31 @@ export class SideMenuComponent {
             addon: 'active'
         };
 
-        this.keymaps$ = store.let(getKeymapEntities());
-        this.macros$ = store.let(getMacroEntities());
+        this.keymaps$ = store.let(getKeymapEntities())
+            .map((keymaps: Keymap[]) => {
+                return keymaps.sort((first: Keymap, second: Keymap) => {
+                    if (first.name > second.name) {
```

Comment:
I have talked with László about it. We should support the local alphabet, because we want localize the agent in the future, so it is better if we use local compare now.